src/compass/utils/age.py

```diff
@@ -493,6 +499,45 @@ def get_cslc_epsg(cslc_file, mission_id='S1', pol='VV'):
     return epsg
 
 
+def get_snr_peak(img: np.ndarray, cutoff_percentile: float=3.0):
+    '''
+    Estimate the signal-to-noise ration (SNR) of the peak
+    in the input image patch
+
+    Parameter
+    ---------
+    img: numpy.ndarray
+        SLC image patch to calculate the SNR
+    cutout: float
+        Cutout ratio of high and low part of the signal to cutoff
+
+    Returns
+    -------
+    snr_peak_db: float
+        SNR of the peak in decibel (db)
```

Comment:
I would say of the CR

Reply:
essentially the function is raking a window of the data and returns the ratio of the max power relative to the average power in the window. So it is the SNR of the peak. isn't it?

Reply:
@hfattahi @vbrancat I think both of your suggestion makes sense, but here I personally would prefer to say `snr_cr` because the peak is the "signal" of corner reflectors.